rdrf/rdrf/workflows/verification.py

```diff
@@ -146,10 +146,10 @@ def has_annotation(self, user, registry_model, patient_model, context_model):
         the cde has changed?
         """
         def carp(msg):
-            logger.debug("Annotations Patient %s Context %s CDE %s: %s" % (patient_model,
-                                                                           context_model.id,
-                                                                           self.cde_model.code,
-                                                                           msg))
+            logger.info("Annotations Patient %s Context %s CDE %s: %s" % (patient_model,
```

Comment:
better to use patient_model.id

Reply:
hum not sure how you ended up seen this line... I must have created the pull request too early and pushed a change later as it has been fixed https://github.com/muccg/rdrf/pull/998/files#diff-a751215b7458437a4864aa8d56b3917fR149 -